### PR TITLE
Updated Social Loss, Social Gain, Social Worse, Social Approach vignettes

### PR DESCRIPTION
**Pull request by mackgalante**
Changed formatting in message templates subsection to reflect updates to other vignettes, fixed a spelling error or two, altered formatting of moderators and preconditions sections for readability.

Note: Not sure if we want to fold in the formatting changes I developed for the message template section into this vignette (since it is essentially the vignette template), but thought I would do the work now and leave it here in a separate branch anyway. If we do like the format, great! Otherwise I can change the other vignettes to match this one.

### CODE REVIEW

**Comment by mackgalante** [comment]
Format changes reflect what we approved at 5/23/2023 meeting, would love to see this branch merged so the template stays up-to-date :)

File: vignettes/social_loss_vignette/readme.md [hunk 1]
```diff
@@ -80,73 +80,75 @@ This pathway describes the influence of feedback interventions that show the mes
...
 ## Message templates
 Message templates represent a possible motivational message that a precision feedback system can send. There are two message templates that the precision feedback system can access for this vignette: 
-
-A. The [Drop Below Peer Average](https://github.com/Display-Lab/knowledge-base/blob/main/message_templates/drop_below_peer_average.json) message template contains the following message: "Your performance dropped below the peer average for the measure [measure name]". This message template [is about] (http://purl.obolibrary.org/obo/IAO_0000136) the following features of performance data:
-1. Social comparator element (http://purl.obolibrary.org/obo/psdo_0000045)
-2. Negative performance gap set (http://purl.obolibrary.org/obo/psdo_0000116)
-3. Negative performance trend set (http://purl.obolibrary.org/obo/psdo_0000119)
-4. Loss set (http://purl.obolibrary.org/obo/psdo_0000122)
-5. Peer average comparator (http://purl.obolibrary.org/obo/PSDO_0000126)
-6. Display format compatibility: Line graph (http://purl.obolibrary.org/obo/IAO_0000573), bar chart (http://purl.obolibrary.org/obo/STATO_0000166)
```
Comment (zachll) [inline vignettes/social_loss_vignette/readme.md:120]: new line needed here

File: vignettes/social_loss_vignette/readme.md [hunk 2]
```diff
@@ -80,73 +80,75 @@ This pathway describes the influence of feedback interventions that show the mes
...
+1. [Negative performance gap set](https://bioportal.bioontology.org/ontologies/PSDO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPSDO_0000116)
+2. [Negative performance trend set](https://bioportal.bioontology.org/ontologies/PSDO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPSDO_0000119)
+3. [Social comparator element](https://bioportal.bioontology.org/ontologies/PSDO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPSDO_0000045) 
+4. [Loss set](https://bioportal.bioontology.org/ontologies/PSDO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPSDO_0000122)
+5. [peer average comparator](https://bioportal.bioontology.org/ontologies/PSDO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPSDO_0000126)
+Display format compatability: [Line Graph](https://bioportal.bioontology.org/ontologies/IAO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FIAO_0000573), [Bar Chart](https://bioportal.bioontology.org/ontologies/STATO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FSTATO_0000166)
```
Comment (zachll) [inline vignettes/social_loss_vignette/readme.md:122]: new line needed here - or #6 maybe

File: vignettes/social_loss_vignette/readme.md [hunk 3]
```diff
@@ -80,73 +80,75 @@ This pathway describes the influence of feedback interventions that show the mes
...
+1. [Negative performance gap set](https://bioportal.bioontology.org/ontologies/PSDO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPSDO_0000116)
+2. [Negative performance trend set](https://bioportal.bioontology.org/ontologies/PSDO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPSDO_0000119)
+3. [Social comparator element](https://bioportal.bioontology.org/ontologies/PSDO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPSDO_0000045) 
+4. [Loss set](https://bioportal.bioontology.org/ontologies/PSDO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPSDO_0000122)
+5. [peer 90th percentile comparator](https://bioportal.bioontology.org/ontologies/PSDO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPSDO_0000129)
+Display format compatability: [Line Graph](https://bioportal.bioontology.org/ontologies/IAO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FIAO_0000573), [Bar Chart](https://bioportal.bioontology.org/ontologies/STATO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FSTATO_0000166)
```
Comment (zachll) [inline vignettes/social_loss_vignette/readme.md:132]: same as above: new line needed here - or #6 maybe

File: vignettes/social_loss_vignette/readme.md [hunk 4]
```diff
@@ -3,13 +3,13 @@
...
 
-This vignette also contains example data and unique identifiers for the features of data and other entities that a precision feedback system uses to reason about the potential success of a precision feedback message. An example of a unique identifier is "http://purl.obolibrary.org/obo/psdo_0000126" which reprsents a peer average comparator, and which is defined in the [Performance Summary Display Ontology](https://github.com/Display-Lab/psdo) as an average representing the mean performance of a peer group. 
+This vignette also contains example data and unique identifiers for the features of data and other entities that a precision feedback system uses to reason about the potential success of a precision feedback message. An example of a unique identifier is a [peer average comparator](https://bioportal.bioontology.org/ontologies/PSDO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPSDO_0000126)" which represents a peer average comparator, and which is defined in the [Performance Summary Display Ontology](https://github.com/Display-Lab/psdo) as an average representing the mean performance of a peer group.
```
Comment (zachll) [inline vignettes/social_loss_vignette/readme.md:6]: remove quotation mark from this line

**Comment by zachll** [comment]
looking good with minor changes, I've added 3 comments in the file

File: vignettes/social_loss_vignette/readme.md [hunk 5]
```diff
@@ -3,13 +3,13 @@
...
 
-This vignette also contains example data and unique identifiers for the features of data and other entities that a precision feedback system uses to reason about the potential success of a precision feedback message. An example of a unique identifier is "http://purl.obolibrary.org/obo/psdo_0000126" which reprsents a peer average comparator, and which is defined in the [Performance Summary Display Ontology](https://github.com/Display-Lab/psdo) as an average representing the mean performance of a peer group. 
+This vignette also contains example data and unique identifiers for the features of data and other entities that a precision feedback system uses to reason about the potential success of a precision feedback message. An example of a unique identifier is a [peer average comparator <sub>(BP)</sub>](https://bioportal.bioontology.org/ontologies/PSDO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPSDO_0000126) which represents a peer average comparator, and which is defined in the [Performance Summary Display Ontology <sub>(GH)</sub>](https://github.com/Display-Lab/psdo) as an average representing the mean performance of a peer group.
```
Comment (zachll) [inline vignettes/social_loss_vignette/readme.md:6]: In this case we want to show the IRI, instead of the label, so that the sentence is not redundant

**Pull request merged**